### DISCUSSION TITLE
How to reset tamper (Lidl HG06336)

### DISCUSSION THREAD

**Discussion by michal-bartak** [Koenkk/zigbee2mqtt#11089] [Devices]
Hello
During looking for best positioning of this door sensor I tampered it accedently. Once tampered, the device is reporting Open and Tampered states.
Unfortunatelly I found no easy way to reset the tamper state. I had to remove the device from Z2M, reset device and re-add it.

While tamper itself is exceptional event, you have to tamper device in order to remove devices. From this POV it seems to me awkward to have to re-add the device to z2m everytime I changed batteries.

So question is: is there any way to reset tamper state of Lidl Door Sensor HG06336?

With regards

**Comment by radiergimmy** [comment]
I've also noticed that the tamper alert gets triggered and can't be reset when replacing the batteries.

**Comment by marvin1309** [comment]
Same Problem. Repairing fixed it for short time but know i broke the reset button :D. There a way to reset Temper from Zigbee2Mqtt? ;)

**Comment by danielfaust** [comment]
Can confirm. Removing it from Zigbee2mqtt and then re-adding it (and pushing a needle in the hole for ~5 seconds) resets the tamper.

**Comment by jo8be** [comment]
I can confirm the problem. 
I have coneccted the switch to Home Assistant.

Looking at the Exposes tab of the device under Zigbee2MQTT:

Once configured it says:
Contact: Open
Tamper: Clear

With magnet it goes to:
Contact: Closed
Tamper: Clear

Removing the mounting plate:
Contact: Open
Tamper: Tampered

With magnet and removed plate:

Contact: Closed
Tamper: Tampered

Mounting tha plate again:

Contact: Open
Tamper: Clear

With magnet it goes again to:
Contact: Closed
Tamper: Clear

Removing the batteries for 30 sec and re-inserting:

Device lost connection. Only deleting it from HA and reconnecting it will help. 

I also did a test with fixing the tamper switch with tape while removing/readding the batteries, but that did not change the behavior.

My conlclusion: 
1. The problem comes from the loss of supply voltage, not from the tamper switch.
2. The device is garbage. That is probably the reason, why it is no longer available at Lidl.
